src/pages/ReportParticipantsPage.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import _ from 'underscore';
+import {
+    View,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import styles from '../styles/styles';
+import ONYXKEYS from '../ONYXKEYS';
+import HeaderWithCloseButton from '../components/HeaderWithCloseButton';
+import Navigation from '../libs/Navigation/Navigation';
+import ScreenWrapper from '../components/ScreenWrapper';
+import OptionsList from '../components/OptionsList';
+import ROUTES from '../ROUTES';
+
+const propTypes = {
+    /* Onyx Props */
+    // The personal details of the person who is logged in
+    personalDetails: PropTypes.shape({
```

Comment:
We have `personalDetailsPropType.js` which you can import instead of redefining the props here.